src/main/resources/com/google/api/codegen/nodejs/version_index.snip

```diff
@@ -49,10 +49,28 @@
     {@comments(@service.doc.lines)}
     *
     * @@param {object=} options - [Configuration object](#/docs).
+    * @@param {string=} options.key - API key to use form communication with the
+    *     service.
+    * @@param {string=} options.keyFilename - Path to a key file to use for an
```

Comment:
Use the wording for relevant parameters here: https://googlecloudplatform.github.io/google-cloud-node/#/docs/google-cloud/0.56.0/google-cloud.